app/components/shared/Icon/index.js

```diff
@@ -35,6 +35,8 @@ const pathNodes = (icon) => {
       return require("./github").default;
     case 'twitter':
       return require("./twitter").default;
+    case 'down-triangle':
+      return require("./down-triangle").default;
```

Comment:
huh! this can actually totally be done with webpack wildcards! maybe I’ll hack that in eventually